### DISCUSSION TITLE
Linkerd installation on OKD

### DISCUSSION THREAD

**Discussion by rishiravikumar-tul-scm** [linkerd/linkerd2#6760] [Q&A]
Hi,
running version stable-2.10.2

I have an OKD cluster with calico CNI, and have installed linkerd CNI. On installing the control plane, the deployments shows errors

Errors:
- Readiness probe failed: HTTP probe failed with statuscode: 503

linkerd check output:
pod/linkerd-controller-66c49c8bb4-rqjwf container public-api is not ready

Any help would be appreciated!

**Comment by cpretzer** [comment]
@RishiRavikumar we'd love some help with getting a documented configuration for OKD. Please have a look at #4851.
